content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+
+## Setup Repo Info
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+*See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
+
+## Installing the Components
+
+To make it easier to install the components, we have created some Helm value files for you. Make sure you have
+pulled the Helm chart package, or download the [latest version of Istio](/docs/setup/getting-started/#download).
+There will be `ambient-values.yaml` files in the chart package.
+
+In the following steps, we will use the downloaded Istio release package as an example, and make sure you are in the
+root directory of the Istio release package.
+
+### Installing Base Component
+
+The **Base** chart contains the basic CRDs and cluster roles required to set up Istio.
+This should be installed prior to any other Istio component.
+
+{{< text bash >}}
+$ helm install istio-base istio/base
+{{< /text >}}
+
+### Installing CNI Component
+
+The **CNI** chart installs the Istio CNI Plugin. It is responsible for detecting the pods that belong to the ambient mesh,
+and configuring the traffic redirection between the ztunnels - which will be installed later.
+
+{{< text bash >}}
+$ helm install istio-cni istio/cni -n kube-system \
+  -f manifests/charts/istio-cni/ambient-values.yaml
```

Comment:
```suggestion
  -f manifests/charts/istio-cni/ambient-values.yaml
```

This specific `values.yaml` only has about 3 values overrides from the base case/default.

In terms of docs, my rule of thumb is 

1. If your `custom-values.yaml` override overrides 3 or fewer fields, docs should use individual `--set` flags and not a `-f file.yaml` flag.

2. If your `custom-values.yaml` override overrides 4 or more fields, docs should use a `-f file.yaml` flag and not individual `--set` flags. Also, if this is a common/base case, you might need to consider if you really need users to explicitly override that many things for the base case, versus implicitly setting some of the values based on other values, as a followup.